loader/src/main/java/net/neoforged/neoforgespi/language/IModLanguageLoader.java

```diff
@@ -35,6 +35,11 @@ public interface IModLanguageLoader {
      */
     String name();
 
+    /**
+     * @return the version of this loader
```

Comment:
```suggestion
     * {@return the version of this loader}
```

Reply:
That *is the correct* way of rendering inline return statements. They act as both a `@return` tag and as a `Returns (...).` description.